src/components/editor/leftNav/ActivePanelPropertyNav.jsx

```diff
@@ -21,6 +20,8 @@ const ActivePanelPropertyNav = (props) => {
     selectPropertyTemplate(state, property?.propertyTemplateKey)
   )
 
+  const handleClick = useLeftNav(property)
+
```

Comment:
our new hook defined below that will prevent clicks from being activated when a modal is open